### PR TITLE
emc flareos update

### PR DESCRIPTION
**Pull request by crcro**
changed the display of EMC CLARiiON to FLARE OS for OS

### CODE REVIEW

File: includes/definitions.inc.php [hunk 1]
```diff
@@ -1058,7 +1058,7 @@
 
 // EMC FlareOS
 $os = 'flareos';
-$config['os'][$os]['text'] = 'EMC CLARiiON';
+$config['os'][$os]['text'] = 'FLARE OS';
```
Comment (laf) [inline includes/definitions.inc.php]: EMC FLARE OS maybe?

Comment (crcro) [inline includes/definitions.inc.php]: as far as I know the official name is FLARE OS (Fibre Logic Array Runtime Environment OS).

Comment (laf) [inline includes/definitions.inc.php]: That's fine for the os name, this is just a descriptive piece of text so EMC FLARE OS (or EMC Flare OS) would be better.

**Comment by crcro** [comment]
changed the description to EMC Flare OS

**Comment by crcro** [comment]
@laf ... is there anyway to make graphs on a different host (like remote agent) and put them to another host, ie:

server running naviseccli to get data out EMC hardware that are not exposed via snmp (like cache and cpu usage) and allocate that graphs to the real emc host in librenms?

**Comment by laf** [comment]
Munin script, you can write your own, get it to do whatever you want like connecting to a remote box and draw graphs on that.

**Pull request merged**